_rules/meta-viewport-b4f0c3.md

```diff
@@ -0,0 +1,120 @@
+---
+id: b4f0c3
+name: meta viewport
+rule_type: atomic
+description: |
+  This rule checks that the meta element is not used to block the user agent ability to zoom. 
+accessibility_requirements:
+  wcag20:1.4.10: # Reflow (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to every `<meta name="viewport">` elements with a `content` attribute in a document.
```

Comment:
I think "document" should be `[document](https://www.w3.org/TR/dom/#concept-document)`.
(but that link seems to only be here on very few rules, so I'm not sure about it…)

Reply:
Do we really need the "in a document" part? The input aspects already specify that this applies to the DOM tree.